Sources/SwiftGen/main.swift

```diff
@@ -23,21 +24,37 @@ let main = Group {
     }
   }
   $0.group("config", "manage and run configuration files") {
-    $0.addCommand("lint", "Lint the configuration file", ConfigCLI.lint)
-    $0.addCommand("run", "Run commands listed in the configuration file", ConfigCLI.run)
-    $0.addCommand("init", "Create an initial configuration file", ConfigCLI.create)
-    $0.addCommand("doc", "Open the documentation for the configuration file on GitHub", ConfigCLI.doc)
+    $0.addCommand("lint", "lint the configuration file", ConfigCLI.lint)
+    $0.addCommand("run", "run commands listed in the configuration file", ConfigCLI.run)
+    $0.addCommand("init", "create an initial configuration file", ConfigCLI.create)
+    $0.addCommand("doc", "open the documentation for the configuration file on GitHub", ConfigCLI.doc)
   }
 
-  $0.group("templates", "manage custom templates") {
+  $0.group("template", "manage custom templates") {
     $0.addCommand("list", "list bundled and custom templates", TemplatesCLI.list)
     $0.addCommand("which", "print path of a given named template", TemplatesCLI.which)
     $0.addCommand("cat", "print content of a given named template", TemplatesCLI.cat)
+    $0.addCommand("doc", "open the documentation for templates on GitHub", TemplatesCLI.doc)
+  }
+
+  $0.group("run", "run individual parser commands without a config file") {
+    for cmd in ParserCLI.allCommands {
+      $0.addCommand(cmd.name, cmd.description, cmd.command())
+    }
+  }
+
+  // Deprecated: Remove this in SwiftGen 7.0
+  $0.group("templates", "DEPRECATED - old spelling for the `template` subcommand") {
+    $0.addCommand("list", "list bundled and custom templates", TemplatesCLI.list)
+    $0.addCommand("which", "print path of a given named template", TemplatesCLI.which)
+    $0.addCommand("cat", "print content of a given named template", TemplatesCLI.cat)
+    $0.addCommand("doc", "open the documentation for templates on GitHub", TemplatesCLI.cat)
```

Comment:
Note: I know I could have mutualised those 4 `$0.addCommand` lines with the ones in `$0.group("template")` on line 33, but I thought it was probably not worth creating a helper function to mutualise those 4 entries just so we undo it all in SwiftGen 7 when we remove the deprecated `templates` group entirely